contributing-guides/style-guide.md

```diff
@@ -104,6 +104,13 @@ Example:
 ```
 - Pre-translated alias page templates can be found [here](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md).
 
+## Option syntax
+
+- Use GNU-style **long options** (like `--help`) when they are portable (available on more than one platform tldr has pages for:
+  `android`, `common`, `linux`, `osx`, `sunos`, `windows`). It means that for all pages in `common/` you can prefer long options
+  when available for everything.
```

Comment:
```suggestion
- Use GNU-style **long options** (like `--help` rather than `-h`) when they are cross-platform compatible (available on multiple platforms among `android`, `linux`, `osx`, `sunos`, `windows`). For pages in `common/`, long options are preferred when available.
```